### ISSUE TITLE
Double tapping opened drawer causes jerky animation.

### ISSUE THREAD

**Issue by realjamie** [root-two/react-native-drawer#84]
When acceptTap is set to true and you double click on the opened Drawer, it will open/close the Drawer quickly several times.

**Comment by natorojr** [comment]
Yes, I see this behavior also. Setting acceptDoubleTap explicitly to `false` does not help either.

It's fairly easy to reproduce even in your sample app RNDrawerDemo. Just enable `acceptTap`, set `panCloseMask` = 1 and `panOpenMask` = 0 (for purposes of reproducibility), open the drawer, then double tapping any where on the Drawer will cause a jerky animation.

**Comment by rt2zz** [comment]
Perhaps a simple and elegant solution is to set a cool down on tap interactions. So e.g. if a tap triggers a toggle, no further taps will be accepted for N ms.

**Comment by realjamie** [comment]
@rt2zz do you think it's a good idea to leverage the tweening ratio? If the ratio is between 0 and 1, disable any gesture handler.

**Comment by rt2zz** [comment]
If need be we can disable all interaction during tween, but I would like to exhaust other options first.

I believe there is a valid use case where a user actually wants to grab the drawer mid-tween.
